example/e2ee.js

```diff
@@ -60,73 +68,91 @@ async function connect()
 		forceEncodedAudioInsertableStreams: true,
 		encodedInsertableStreams: true
 	});
-	
+
 	const senderId = 0;
 	const receiverId = 1;
-	
+
 	//Create contexts
 	const senderClient   = await SFrame.createClient(senderId, {
 		skipVp8PayloadHeader : true
 	});
 	const receiverClient = await SFrame.createClient(receiverId, {
 		skipVp8PayloadHeader : true
 	});
-	
+
 	await senderClient.setSenderEncryptionKey(shared);
 	await senderClient.setSenderSigningKey(keyPair.privateKey);
-	
+
 	await receiverClient.addReceiver(senderId);
 	await receiverClient.setReceiverEncryptionKey(senderId,shared);
+
+	senderPasswordForm.addEventListener('submit', async (e) =>  {
+		e.preventDefault();
+		const newKey = await getRoomKey("roomId", senderPasswordInput.value);
```

Comment:
Nit.  Same here re: use of double quotes.  Probably elsewhere in the code as well.